### PR TITLE
fix warnings

### PR DESCRIPTION
**Pull request by njarecha**
Screenshot 2023-04-18 at 2 24 39 PM

### CODE REVIEW

**Review by mhdostal — APPROVED** [review]
@njarecha Thank you!

**Review by rolson — DISMISSED** [review]

**Comment by njarecha** [comment]
@rolson @mhdostal The AuthenticationExample was set to `Mac (Designed for iPad)` instead of `Mac (Mac Catalyst)` so fixed it. Please re-review.

File: AuthenticationExample/AuthenticationExample/AuthenticationExample.entitlements [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.security.app-sandbox</key>
+	<true/>
+	<key>com.apple.security.network.client</key>
```
Comment (rolson) [inline AuthenticationExample/AuthenticationExample/AuthenticationExample.entitlements:7]: is this new entitlement necessary?

Comment (njarecha) [reply AuthenticationExample/AuthenticationExample/AuthenticationExample.entitlements:7]: It was automatically added by Xcode when I added the destination

**Review by rolson — APPROVED** [review]

File: AuthenticationExample/AuthenticationExample/AuthenticationExample.entitlements (hunk 1 above)
Comment (njarecha) [reply AuthenticationExample/AuthenticationExample/AuthenticationExample.entitlements:7]: It is there for Example app as well
https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/blob/main/Examples/Examples.xcodeproj/project.pbxproj#L60

**Comment by njarecha** [comment]
Thanks @rolson and @mhdostal!

**Pull request merged**